tools/tkn-results/README.md

```diff
@@ -0,0 +1,13 @@
+# tkn-results
+
+tkn CLI plugin for interacting with the Tekton Results API.
+
+## Installation
+
+```
+$ GOBIN=${TKN_PLUGINS_DIR:-"${HOME}/.config/tkn/plugins"} go install github.com/tektoncd/experimental/tkn-resolve
```

Comment:
Is `github.com/tektoncd/experimental/tkn-resolve` the right path?

Reply:
Good catch! 😅  Fixed.